tools/shared.py

```diff
@@ -2499,7 +2499,7 @@ def closure_compiler(filename, pretty=True):
         exports_file.write(module_exports_suppressions.encode())
         exports_file.close()
 
-        args.append('--js')
+        args.append('--externs')
```

Comment:
this change seems surprising in this PR?

Reply:
This is actually a bugfix, the DECLARE_ASM_MODULE_EXPORTS=0 exported symbols need to be declared extern so that Closure does not minify them.

Reply:
Ah,if this change `js => externs` is a bugfix, then please open a separate PR for it, before this one?